packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -2252,8 +2287,10 @@ class SizedOverflowBox extends SingleChildRenderObjectWidget {
 ///
 /// See also:
 ///
-///  * The [catalog of layout widgets](https://flutter.io/widgets/layout/).
+///  * [Visibility], which can hide a child more efficiently (albeit less
+///    subtly).
```

Comment:
Why less subtly?

Reply:
Added: "because it is either visible or hidden, rather than allowing fractional opacity values"